pymc3/model.py

```diff
@@ -914,7 +938,7 @@ def __init__(self, type=None, owner=None, index=None, name=None, data=None,
             data = pandas_to_array(data)
             type = TensorType(distribution.dtype, data.shape)
 
-        super(TensorVariable, self).__init__(type, None, None, name)
+        super(ObservedRV, self).__init__(type, owner, index, name)
```

Comment:
Seems like all is OK https://github.com/ferrine/pymc3/blob/90ab385b48fa4ab9e58901befd86b25af00f1218/pymc3/model.py#L941